README.md

```diff
@@ -172,6 +209,18 @@ No, signing a team up to be a no-FF team includes some basic education on the pr
 participants are aware the team is moving to that model.  If you'd like to sign your team up, please speak with Gina Hargan who will
 be happy to help on-board your team.
 
+## vendoring generated manifests into other repositories
+If your repository relies on vendoring and copying CRD manifests (good job!), you'll need have an import line that
+depends on the package that contains the CRD manifests.
+For example, adding
+```go
+import (
+	_ "github.com/openshift/api/operatoringress/v1/zz_generated.crd-manifests"
+)
+```
+
+Once added, a `go mod vendor` will pick up the package containing the manifests for you to copy.
```

Comment:
Maybe worth mentioning that using `tools/tools.go` for these sort of build-time imports seems to be a common convention.